code/modules/mob/living/carbon/xenomorph/damage_procs.dm

```diff
@@ -144,6 +144,19 @@
 	if(updating_health)
 		updatehealth()
 
+/mob/living/carbon/xenomorph/proc/apply_healing(amount = 0, damagetype = BRUTE, updating_health = FALSE)
+	SEND_SIGNAL(src, COMSIG_XENOMORPH_HEALED_BY_ABILITY, amount, damagetype)
+
+	switch(damagetype)
+		if(BRUTE)
+			adjustBruteLoss(-amount)
+		if(BURN)
+			adjustFireLoss(-amount)
+
+	if(updating_health)
+		updatehealth()
+	return amount
+
```

Comment:
Move undef HANDLE OVERHEAL if you arent using it

Reply:
Done in next commit.